modules/auxiliary/scanner/http/trace.rb

```diff
@@ -27,36 +31,36 @@ def run_host(target_host)
     begin
       res = send_request_raw({
         'version'      => '1.0',
-        'uri'          => '/',
+        'uri'          => '/<script>alert(1337)</script>', #XST Payload
```

Comment:
The alerted text should be selected random as this is quite easy to be fingerprinted by the IDS/AV.